lib/Group_LDAP.php

```diff
@@ -1001,6 +1001,33 @@ public function groupExists($gid) {
 		return true;
 	}
 
+	public function getGroupDetails($gid) {
```

Comment:
doc missing, and in fact I would appreciate the explanation.

Reply:
Not documented in the abstract class nor in the interface :facepalm: 
I'll open a ticket in core to document this where it should be. I'm adding the docs here too.